diff --git a/epan/frame_data.h b/epan/frame_data.h
index d4b429b596..ae992f9c91 100644
--- a/epan/frame_data.h
+++ b/epan/frame_data.h
@@ -89,7 +89,6 @@ typedef struct _frame_data {
   unsigned int need_colorize    : 1; /**< 1 = need to (re-)calculate packet color */
   unsigned int tsprec           : 4; /**< Time stamp precision -2^tsprec gives up to femtoseconds */
   nstime_t     abs_ts;       /**< Absolute timestamp */
-  nstime_t     shift_offset; /**< How much the abs_tm of the frame is shifted */
   guint32      frame_ref_num; /**< Previous reference frame (0 if this is one) */
   guint32      prev_dis_num; /**< Previous displayed frame (0 if first one) */
 } frame_data;
diff --git a/epan/dissectors/packet-frame.c b/epan/dissectors/packet-frame.c
index fdf2ea2ee2..526133dd75 100644
--- a/epan/dissectors/packet-frame.c
+++ b/epan/dissectors/packet-frame.c
@@ -47,7 +47,6 @@ static int proto_pkt_comment = -1;
 static int proto_syscall = -1;
 
 static int hf_frame_arrival_time = -1;
-static int hf_frame_shift_offset = -1;
 static int hf_frame_arrival_time_epoch = -1;
 static int hf_frame_time_delta = -1;
 static int hf_frame_time_delta_displayed = -1;
@@ -476,9 +475,6 @@ dissect_frame(tvbuff_t *tvb, packet_info *pinfo, proto_tree *parent_tree, void*
 								  " the valid range is 0-1000000000",
 								  (long) pinfo->abs_ts.nsecs);
 			}
-			item = proto_tree_add_time(fh_tree, hf_frame_shift_offset, tvb,
-					    0, 0, &(pinfo->fd->shift_offset));
-			proto_item_set_generated(item);
 
 			if (generate_epoch_time) {
 				proto_tree_add_time(fh_tree, hf_frame_arrival_time_epoch, tvb,
@@ -829,11 +825,6 @@ proto_register_frame(void)
 		    FT_ABSOLUTE_TIME, ABSOLUTE_TIME_LOCAL, NULL, 0x0,
 		    "Absolute time when this frame was captured", HFILL }},
 
-		{ &hf_frame_shift_offset,
-		  { "Time shift for this packet", "frame.offset_shift",
-		    FT_RELATIVE_TIME, BASE_NONE, NULL, 0x0,
-		    "Time shift applied to this packet", HFILL }},
-
 		{ &hf_frame_arrival_time_epoch,
 		  { "Epoch Time", "frame.time_epoch",
 		    FT_RELATIVE_TIME, BASE_NONE, NULL, 0x0,
diff --git a/epan/frame_data.c b/epan/frame_data.c
index 4ccd318b9a..cf29e7a49c 100644
--- a/epan/frame_data.c
+++ b/epan/frame_data.c
@@ -202,8 +202,6 @@ frame_data_init(frame_data *fdata, guint32 num, const wtap_rec *rec,
   fdata->has_user_comment = 0;
   fdata->need_colorize = 0;
   fdata->color_filter = NULL;
-  fdata->shift_offset.secs = 0;
-  fdata->shift_offset.nsecs = 0;
   fdata->frame_ref_num = 0;
   fdata->prev_dis_num = 0;
 }
diff --git a/ui/CMakeLists.txt b/ui/CMakeLists.txt
index f430bb079b..79891032d3 100644
--- a/ui/CMakeLists.txt
+++ b/ui/CMakeLists.txt
@@ -51,7 +51,6 @@ set(NONGENERATED_UI_SRC
 	tap-rlc-graph.c
 	tap-tcp-stream.c
 	text_import.c
-	time_shift.c
 	traffic_table_ui.c
 	util.c
 	voip_calls.c
