diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4750b28633..036809fd61 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,7 +23,7 @@ else()
 	set(_project_name wireshark)
 endif()
 
-project(${_project_name} C CXX)
+project(${_project_name} C)
 
 # Updated by tools/make-version.pl
 set(GIT_REVISION 0)
@@ -776,9 +776,6 @@ foreach(THIS_FLAG ${WIRESHARK_COMMON_FLAGS} ${WIRESHARK_C_ONLY_FLAGS})
 endforeach()
 set(CMAKE_C_FLAGS "${ADDED_CMAKE_C_FLAGS} ${CMAKE_C_FLAGS}")
 
-foreach(THIS_FLAG ${WIRESHARK_COMMON_FLAGS} ${WIRESHARK_CXX_ONLY_FLAGS})
-	test_compiler_flag(CXX ${THIS_FLAG} ADDED_CMAKE_CXX_FLAGS)
-endforeach()
 set(CMAKE_CXX_FLAGS "${ADDED_CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS}")
 
 # Strips the source and build directory prefix from the __FILE__ macro to ensure
@@ -791,7 +788,6 @@ if(CMAKE_C_COMPILER_ID MATCHES "GNU" OR CMAKE_C_COMPILER_ID MATCHES "Clang")
 	string(REGEX REPLACE "/$" "" _relative_source_dir "${_relative_source_dir}")
 
 	check_c_compiler_flag(-fmacro-prefix-map=old=new C_fmacro_prefix_map_old_new_VALID)
-	check_cxx_compiler_flag(-fmacro-prefix-map=old=new CXX_fmacro_prefix_map_old_new_VALID)
 	foreach(_lang C CXX)
 		if(${_lang}_fmacro_prefix_map_old_new_VALID)
 			set(_flags CMAKE_${_lang}_FLAGS)
